crypto/ec/build.info

```diff
@@ -50,7 +50,7 @@ $COMMON=ec_lib.c ecp_smpl.c ecp_mont.c ecp_nist.c ec_cvt.c ec_mult.c \
         ecdsa_ossl.c ecdsa_sign.c ecdsa_vrf.c curve25519.c \
         curve448/arch_32/f_impl.c curve448/f_generic.c curve448/scalar.c \
         curve448/curve448_tables.c curve448/eddsa.c curve448/curve448.c \
-        $ECASM ec_backend.c ecx_backend.c
+        $ECASM ec_backend.c ecx_backend.c ecp_secp384r1.c
```

Comment:
We should probably add this file conditionally, e.g., on `s390x` this shouldn't be included.

Reply:
Thanks for the tip! That seems like an easy fix.

Reply:
addressed in 5fc91a34cb069928b0ebf5e08e4ee155980e190f